mls_validation_service/Cargo.toml

```
@@ -34,9 +34,9 @@ xmtp_proto = { path = "../xmtp_proto", features = ["proto_full", "convert"] }
 anyhow.workspace = true
 ethers.workspace = true
 rand = { workspace = true }
+xmtp_common = { workspace = true, features = ["test-utils"] }
 xmtp_id = { workspace = true, features = ["test-utils"] }
```

With the new feature we added below.
```suggestion
xmtp_id = { workspace = true, features = ["test-utils", "scw_cache"] }
```